cloud-sql/mysql/sqlalchemy/README.md

```diff
@@ -84,24 +84,24 @@ for more details on connecting a Cloud Run service to Cloud SQL.
 1. Build the container image:
 
 ```sh
-gcloud builds submit --tag gcr.io/[YOUR_PROJECT_ID]/run-sql
+gcloud builds submit --tag gcr.io/[YOUR_PROJECT_ID]/run-mysql
 ```
 
 2. Deploy the service to Cloud Run:
 
 ```sh
-gcloud beta run deploy run-sql --image gcr.io/[YOUR_PROJECT_ID]/run-sql
+gcloud run deploy run-sql --image gcr.io/[YOUR_PROJECT_ID]/run-mysql
```

Comment:
This still says `run-sql`, but below it says `run-mysql`. Is that right?

Reply:
nope! missed that one. Fixed.